app/views/start_page.scala.html

```diff
@@ -76,11 +74,7 @@ <h2 id="report-your-arrival-and-departure" class="heading-medium">@messages("sta
         @messages("startPage.buttonName")
       </a>
 
-      <div>
-        <br/>
-        <br/>
-        <br/>
-        <br/>
+      <div style="margin-top: 5em">
```

Comment:
why inline style ?

Reply:
There was no scss file with layouts but it might be a good time to create one.